### ISSUE TITLE
Tests?

### ISSUE THREAD

**Issue by Nubuck** [Nubotics/generator-arc-static#5]
Mocha, Karma.... Jest?

**Comment by delta1** [comment]
I don't have a preference here since I haven't used any of them! Which way you leaning?

**Comment by delta1** [comment]
oh it seems that Karma is not actually a testing framework but a test runner and can be used with Mocha/Jasmine/Jest

**Comment by Nubuck** [comment]
True, there are other test runners - Karma is kind of the go to. 

I like the way Jest automatically mocks dependencies and I'm already using Jest in Neo - so seems like an easy way to go - bust since its just the generator I'm referring to testing Mocha should be fine too.

Should probably open another issue to add test scaffolding to the generator. Since the generated project is static - i think TDD or BDD is unnecessary - but I think Nightmare test scaffolds getting generated would be sweet.
